netplugin/netd.go

```diff
@@ -456,6 +456,18 @@ func createContainerEpOper(netPlugin *plugin.NetPlugin, contUUID, contName strin
 				log.Errorf("Endpoint creation failed. Error: %s", err)
 				return err
 			}
+
+			err = operEp.Read(epID)
+			if core.ErrIfKeyExists(err) != nil {
```

Comment:
We should not ignore key not found error here. We should write the oper state back only if we found the key